component/admin/controllers/icalevent.php

```diff
@@ -1182,13 +1182,13 @@ private function doSave(& $msg)
 		if ($event = SaveIcalEvent::save($array, $this->queryModel, $rrule))
 		{
 			$row = new jIcalEventRepeat($event);
-			if (JEVHelper::canPublishEvent($row))
+			if (!JEVHelper::canPublishEvent($row) && !$event->state)
```

Comment:
If editor is not a publisher and new state is 0 (from saveIcalEvent) then we say 'under review' but if If editor is not a publisher and new state is 1 (from saveIcalEvent) then we should say 'event saved'.